app/controllers/calendar_controller.rb

```diff
@@ -14,8 +36,31 @@ def index
     @conference_rooms = ConferenceRoom.all
   end
 
+  # Index for showing events from Google calendar
+  def google_index
+    week_start, week_end = build_week_boundaries(params[:date])
+    @days = (week_start..week_end).to_a
+
+    start_time = Time.now.at_beginning_of_day
+    end_time = Time.now.at_end_of_day
+    step = 30.minutes
+    @times = time_interval(start_time, end_time, step)
+
+    @events = GoogleEvent.list_events(session[:credentials], DateTime.now, DateTime.now + 1.days)
+
+    @conference_rooms = ConferenceRoom.all
```

Comment:
here calling `.all`  AR method is ok as there won't be that many conf room, but remember to avoid it if it's called on object that have high number of instances in the database